resources/content/schema/admin/admin-auth.json

```diff
@@ -1,90 +1,134 @@
 {
     "authorize" : {
-        "[a-zA-Z]*" : "crud",
-        "[a-zA-Z]*\\..*" : "r",
-
+    	".*" : "",
+    	
         ".*\\..*Id" : "cru",
```

Comment:
@ibuildthecloud - Do you mean that the *Id fields should have a "r" by default ?